packages/web/components/Landing/landingSection.ts

```diff
@@ -47,7 +47,12 @@ export const sections = [
   },
   {
     label: '08',
+    internalLinkId: 'what-say',
+    title: '8. What people are saying',
```

Comment:
The other titles are punctuated.

```suggestion
    title: '8. What people are saying.',
```